includes/Generator/Plugins/Twitter.php

```diff
@@ -43,10 +38,7 @@ public function __construct() {
 
 		$this->conversions = array_merge(
 			$this->conversions, [
-			'twitter_site' => 'twitter:site',
-			'description' => 'twitter:description',
-			'image' => 'twitter:image',
-			'image_alt' => 'twitter:image:alt'
```

Comment:
I guess as we don't want/support different images for OpenGraph and Twitter, this is okay